packages/beacon-node/src/chain/stateCache/stateContextCache.ts

```diff
@@ -41,21 +41,12 @@ export class StateContextCache {
     }
 
     this.metrics?.hits.inc();
-    // clonedCount + 1 as there's a .clone() below
-    this.metrics?.stateClonedCount.observe(item.clonedCount + 1);
-    if (!stateInternalCachePopulated(item)) {
-      this.metrics?.stateInternalCacheMiss.inc();
-    }
-
-    // Clone first to account for metrics below
-    const itemCloned = item.clone();
-
     this.metrics?.stateClonedCount.observe(item.clonedCount);
```

Comment:
since we don't call `clone()` anymore, should metric observe `clonedCount` here?